src/test/kotlin/VehicleTest.kt

```diff
@@ -6,6 +6,17 @@ import traffic_simulation.Vehicle
 class VehicleTest {
 
     @Test
+
+    fun doesDelayingVehicleWork(){
+        val trafficJam : Boolean =true
+        val BMW : Vehicle = Vehicle(1 ,true)
+        BMW.delayed=false
+        BMW.gettingDelayed()
+
+        assertEquals(trafficJam,BMW.delayed)
```

Comment:
finde es etwas verwirrend dass du hier für den wahren Wert - true - extra den Wert `trafficJam` erschaffen hast. Könntest doch auch einfach in die Prüfung `true` schreiben.

Also vlt. Zeile 11 löschen und Zeile 16 ändern zu:
`assertEquals ( true , BMW,delayed )`